lib/vector-buffers/src/variants/disk_v2/tests/invariants.rs

```diff
@@ -442,11 +523,10 @@ async fn reader_deletes_data_file_around_record_id_wraparound() {
             let deleted_data_file = assertion_registry
                 .build()
                 .with_name("delete_completed_data_file")
-                .with_parent_name("12345")
+                .with_parent_name("reader_deletes_data_file_around_record_id_wraparound")
```

Comment:
Heh, little more clear.

Reply:
😅 

Yeah, I had reduced these to short number sequences when debugging at some point, because otherwise the line had like... 90 - 100 characters before you go to the span name/target name and it was miserable to look at in my terminal.

Still want to figure out a better way to handle segmenting this stuff, since the whole idea is having a unique top-level span name so that tracing assertions can't bleed across tests... but there's gotta be a less-invasive way to accomplish that.